doc/install/install.rst

```diff
@@ -1,296 +1,147 @@
+Linking to executables you already have
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-QIIME is written in Python 2, and requires Python version greater than Python 2.7.0 and less than Python 3.0.0. You can verify which version of Python you're running with::
+If you already have the appropriate version of these programs installed,
+you can ensure that they run the correct executable by making a symbolic
+link in the bin folder of your Conda QIIME environment. To allow your
+environment to access dependency executables, you can put symbolic links
+to them in your environment's bin folder. The path to this folder is
+displayed when you load the environment:
 
-    python --version
+::
 
+    source activate qiime191
+    discarding /home/username/miniconda3/bin from PATH
+    prepending /home/username/miniconda3/envs/qiime191/bin to PATH
 
-If you encounter problems installing QIIME, please visit http://help.qiime.org.
+To make a symbolic link:
 
-Then, just pip install qiime!
------------------------------
+::
 
-The easiest way to install the latest QIIME release and its base dependencies is with pip::
+    ln -s /path/to/myexecutable /home/username/miniconda3/envs/qiime191/bin/myexecutable
 
-	pip install numpy
-	pip install qiime
+Downloading and linking to new executables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you do not have pip, the easiest way to install it is by running::
+If you don’t have the programs installed already, or would like to use
+an install specific to your particular environment, you can make new
+installations directly into your Conda environment directory.
 
-	easy_install pip
+First, it is helpful to create a variable for your Conda environment
+path:
 
-Or, if you are on Ubuntu, run::
+::
 
-  apt-get install python-pip
+    QIIME_ENV=/home/username/miniconda3/envs/qiime191
 
-**Note:** You may need to prefix the above commands with ``sudo`` if you do not have permission to place files in the default locations. You will also need an active Internet connection.
+Next, make a directory for your QIIME dependencies and change to that
+directory:
 
-These commands install QIIME and all of its Python dependencies. ``pip install qiime`` will also attempt to install some non-Python dependencies, including uclust, FastTree, SortMeRNA, SUMACLUST, and swarm. If any of the non-Python dependencies cannot be installed, they will be skipped and you'll need to install them manually if you need to use them. See below for links to where you can obtain these dependencies if any of them fail to install with ``pip``.
+::
 
-Alternatives to pip install qiime
----------------------------------
+    mkdir $QIIME_ENV/qiime_deps
+    cd $QIIME_ENV/qiime_deps
 
-If you don't have permission (or don't want) to install QIIME into your system (e.g., with ``sudo``), you might want to look into these alternatives:
+Download the binaries for the dependency (uclust, in this case), and
+link to the Conda bin (for macOS use curl instead of wget):
```

Comment:
As of 10.12 Sierra, it is called macOS. https://en.wikipedia.org/wiki/OS_X#Version_10.12:_.22Sierra.22 <https://en.wikipedia.org/wiki/OS_X#Version_10.12:_.22Sierra.22>

I will update to curl -sO.

> On Jul 21, 2016, at 2:19 PM, Greg Caporaso <notifications@github.com> wrote:
> 
> In doc/install/install.rst <https://github.com/biocore/qiime/pull/2162#discussion_r71789050>:
> 
> >  
> > -If you don't have permission (or don't want) to install QIIME into your system (e.g., with ``sudo``), you might want to look into these alternatives:
> > +Download the binaries for the dependency (uclust, in this case), and
> > +link to the Conda bin (for macOS use curl instead of wget):
> This should be Mac OS. Also, could you just update all instructions to be curl -sO instead of wget? Since curl is cross-platform, we should just always use that (it's also much more powerful than wget).
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/biocore/qiime/pull/2162/files/1fe89832973abd83a05bee07283a7390cca1832b#r71789050>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AFa_ZunHCnN6Qdii-Z8LlYDf6srIPV64ks5qX-JJgaJpZM4JRRrq>.
>